src/module/lighthouse/index.ts

```diff
@@ -18,12 +18,21 @@ import parseReport from './output';
 
 const lighthouseRunner = async (
   url: string,
-  options: Options,
+  userOptions: Options,
   commandOptions: CommandOptions,
   config: LighthouseConfig = Config.get('configs.lighthouse', defaultConfig),
 ): Promise<Report> => {
-  /* eslint-disable-next-line no-param-reassign  */
-  options.output = ['json'];
+  // Default config options
+  const defaults = {
+    output: ['json'],
+    maxWaitForFcp: 60 * 1000,
+  };
+
+  // Build options but let users change the defaults if needed
+  const options = {
+    ...defaults,
+    ...userOptions,
```

Comment:
What about smashing `config` in here too? The benefit, `options` is now configurable. Downside, there may be extra options passed to lighthouse that will be disregarded and may cause collisions. Could always pick an array of keys off `config` to throw into `options` if you are worried abbout that. This way, a config file would be:

```yaml
configs:
  lighthouse:
    maxWaitForFcp: 120000
```

Reply:
Yes, that'd be fantastic. We'd need to make it universal for all modules.
Also, I'd prefer `maxWaitForFcp` to be larger than the lighthouse default as it seems like a common issue

Reply:
What would be fantastic? Smashing config there blindly or having a way to pick props off the config? Here, you could:

```javascript
const options = {
  ...defaults,
  maxWaitForFcp: Config.get('configs.lighthouse.maxWaitForFcp'),
  ...userOptions,
};
```

or whatever order you'd want. If you want a better api to smash an array of keys, that can be done too.

Reply:
Well I would smash the defaults and load config from the user config file. There's already a backlog item for that effort, I believe.

Reply:
I opened a PR into your branch to describe what I was thinking: #107